acceptance/tests/fixtures/cases/job-client-inject-grace-period-30s/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+
+resources:
```

Comment:
I left a similar comment somewhere else but I think 30 seconds is too long of a wait. Maybe change this to 10.